### PR TITLE
rm alt faucet

### PR DESCRIPTION
**Pull request by wsdt**
Removing dead link to alt faucet.

### CODE REVIEW

File: boba_community/turing-twitter-pay/aws/.aws-sam/build.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# This file is auto generated by SAM CLI build command
```
Comment (InoMurko) [inline boba_community/turing-twitter-pay/aws/.aws-sam/build.toml]: can you remove this file?

File: boba_community/turing-twitter-pay/aws/.aws-sam/build.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+[function_build_definitions]
+[function_build_definitions.b24486e9-3ed4-4851-a5ca-cbbbc97f375e]
+codeuri = "C:\\Users\\Kevin Riedl\\Desktop\\boba\\boba_community\\turing-twitter-pay\\aws"
```
Comment (InoMurko) [inline boba_community/turing-twitter-pay/aws/.aws-sam/build.toml]: you use windows? I am shocked.

File: boba_community/turing-twitter-pay/aws/.aws-sam/build.toml (hunk 1 above)
Comment (wsdt) [reply boba_community/turing-twitter-pay/aws/.aws-sam/build.toml]: fixed

**Review by InoMurko — APPROVED** [review]

File: boba_community/turing-twitter-pay/aws/.aws-sam/build.toml (hunk 2 above)
Comment (wsdt) [reply boba_community/turing-twitter-pay/aws/.aws-sam/build.toml]: Docker & WSL to the rescue

**Pull request merged**